.ci/pytorch/macos-test.sh

```diff
@@ -149,21 +150,95 @@ test_jit_hooks() {
   assert_git_not_dirty
 }
 
+
+test_torchbench_perf() {
+  print_cmake_info
+
+  echo "Launching torchbench setup"
+  make -C $(pwd)/benchmarks/dynamo clone-deps
+  make -C $(pwd)/benchmarks/dynamo pull-deps
+  make -C $(pwd)/benchmarks/dynamo build-deps
```

Comment:
The job is running now https://github.com/pytorch/pytorch/actions/runs/10910203946/job/30287846856#step:15:384, but I think you need to update the the dependency setup to make it work with CI as make -C /Users/ec2-user/runner/_work/pytorch/pytorch/benchmarks/dynamo build-deps fails.  Also, one suggestion is that you don’t need to use steps like creating new conda environment or run conda activate in CI because the job is already running in a conda env that will be clean up after.  So just conda install or pip install your dependencies